recipes/redlionfish/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "RedLionfish" %}
+{% set version = "0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/rosalindfranklininstitute/RedLionfish/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 960140610b16b2209acb91ef5eb14fa24d0b8a3f90022ef6e9445b188e831bcb #v0.5
+
+build:
+  number: 0
+
+  script: "{{ PYTHON }} ./setup_for_condabuild.py -vv install"
```

Comment:
Did you intend to leave this line like this?

Reply:
yes. the setup.py is slightly different from the setup_for_condabuild.py file, in the sense that the latter has less requirements. These requirements are included in the meta.yaml file. setup.py is for PyPI or local installations. I had some issues when I tried to have a single file, reason why I made a different one for the conda based installation.